src/kvirc/module/KviModuleManager.cpp

```diff
@@ -175,8 +175,6 @@ bool KviModuleManager::loadModule(const QString & modName)
 	}
 
 	QLibrary * pLibrary = new QLibrary(tmp);
-	pLibrary->setLoadHints(QLibrary::ExportExternalSymbolsHint);
```

Comment:
Won't this break perl again?

Reply:
Perl actually worked for everybody but some gentoo boxes. I've tested reverting the change and it works for me on both linux and windows,but i didn't test gentoo specifically.
To be honest i'm not even sure the fix worked, since there's no feedback about it.

Reply:
I use gentoo, and I remember kvirc crashing a lot for me while I tried to use perl scripting there :) I don't remember whether it was before or after that change.

Reply:
The original issue was about the perlcore module not loading at all, so as long as an hello world kvs/perl script works, it should be ok.